.env

```diff
@@ -1,3 +1,5 @@
 export PG_DATABASE_URL=postgres://granite:password@localhost:5432/granite_db
 export MYSQL_DATABASE_URL=mysql://granite:password@localhost:3306/granite_db
 export SQLITE_DATABASE_URL=sqlite3:./config/granite.db
+export POSTGRES_IMAGE="${POSTGRES_IMAGE:=postgres:10.4}"
+export MYSQL_IMAGE="${MYSQL_IMAGE:=mysql:5.7}"
```

Comment:
It seems like these variables don't need to be in here since they're in travis.yml

Reply:
Travis.yml will only load on the travis service. If you want to run docker locally it will raise an error that these variables are not in your env. If you source .env the file now it will include the default values into your env so you can use docker locally as well

Reply:
Got it, I see your goal. Thanks for the clarification.

I don't _know_ but I suspect this won't work with the crystal .env loaders (eg #236). That's not necessarily an objection, just something that should be made clear. My preference is to not need to `source .env` files on my dev machine for anything.

Reply:
Do we know how many people develop using the non docker version?  I'd be curious to know.

Reply:
Actually i use it because i have specific databases installed

Reply:
Actually i use it because i have specific databases installed